DEVELOPING.MD

```diff
@@ -65,17 +65,18 @@ Here are some examples:
 
 NOTE: the charts _do not match_ the upstream charts exactly - the `rancher/vsphere-charts` package actually predates those by a lot due to the fact that upstream did NOT ship a chart instead they just shipped raw manifests which isn't very easy to package/update. Engineers used those initial manifests and made these from them.
 
-There are two goals when checking upstream manifests. The first goal is to check to see if any image versions have been updated and the second is to check if any changes in the manifests have occured.
+There are two goals when checking upstream manifests. The first goal is to check to see if any image versions have been updated and the second is to check if any changes in the manifests have occurred.
 
 If you find that image versions have been updated, then you will need to make a pull request to the [image-mirror](https://github.com/rancher/image-mirror) repository first before proceeding. [Here](https://github.com/rancher/image-mirror/pull/261) is an example PR and follow the requirements outlined in the PR template for those changes. After the images have been checked and image-mirror updated with any new images or image versions, then proceed to check check the manifests.
 
 Checking the manifests is best done by doing a tag compare in each upstream chart. Compare an upstream tag to the tag that the rancher chart is based off of by viewing the `Full Changelog` for that tag. This compares the manifest directories. If there is a change that needs to be added to charts in this repository, bring it in. [Here](https://github.com/kubernetes/cloud-provider-vsphere/compare/v1.25.0...v1.24.2) is an example. Just look for important changes and bring them into this chart as necessary. New environment variables, configs, etc.
 
 Make sure that the unit tests for each chart have been updated, or new tests or test cases are added based on the set of manifest changes.
 
-Finally increment the chart version, if necessary. We need to update the chart version anytime that a new tag of CSI/CPI is released and we update to that version of upstream.
+Finally, update the chart’s metadata in the Chart.yaml file. There are two key fields to adjust:
 
-CPI and CSI versioning is different because CSI tracks the version from upstream so it needs a `-rancher1` for when we need to do a Rancher specific patch, whereas CPI doesn’t track the same version from upstream (they have multiple CPIs based on Kubernetes version. Our CPI chart is based on all of them together). We use our own versioning so we can bump the patch for Rancher specific versions. [Here](https://github.com/rancher/vsphere-charts/pull/36/commits/5bb4f6a042dc31ace66522fe670a72930d5d3e63) is an example.
+- `appVersion`: Indicates the latest version of the upstream application used in the chart. Update this whenever a new CSI/CPI tag is incorporated.
+- `version`: Represents the chart’s own version. Update this whenever any change is made to the chart.
```

Comment:
Can we note here how we're using the -rancherX suffix? Or do we want to drop it entirely? The two charts are not consistent about this.